rtlib/src/SepDisegni.cpp

```diff
@@ -0,0 +1,275 @@
+//
+// Created by Sammy Stampley on 11/12/19.
+//
+//
+// Created by Sammy Stampley on 10/3/19.
+//
+
+#include "SepDisegni.h"
+
+//Constructor for the SepDisegni Class
+SepDisegni::SepDisegni(){
+    Mat image_;
+    Mat watershededImage_;
+    vector<Mat> fragmentedImage_;
+}
+
+
+
+//This takes an Input Image and sets it into your private image member
+const void SepDisegni::SetImage(Mat input) {
+    image_ = input;
+}
+
+//This retrieves the array of fragmented images that are stored
+const vector<Mat> SepDisegni::GetFragmentedImage() {
+    return fragmentedImage_;
+}
+
+vector<Mat> SepDisegni::Compute(){
+    watershededImage_ = watershed_image();
+    fragmentedImage_ = split_merge(watershededImage_);
+    return fragmentedImage_;
+}
+
+Mat SepDisegni::watershed_image() {
+
+
+    Mat src = image_;
+
+    // Show source image
+    namedWindow("Source Image", 0);
+    imshow("Source Image", src);
+    waitKey(0);
```

Comment:
All of these operations for showing images were for debugging during development. Remove them all.